gamestonk_terminal/stocks/sector_industry_analysis/financedatabase_model.py

```diff
@@ -0,0 +1,248 @@
+"""Finance Database Model"""
+__docformat__ = "numpy"
+# pylint:disable=too-many-arguments
+
+from tqdm import tqdm
+import financedatabase as fd
+import yfinance as yf
+
+
+def get_countries():
+    """
+    Get all countries in Yahoo Finance data. [Source: Finance Database]
+
+    Returns
+    -------
+    list
+        List of possible countries
+    """
+    return fd.show_options("equities", "countries")
+
+
+def get_sectors():
+    """
+    Get all sectors in Yahoo Finance data. [Source: Finance Database]
```

Comment:
Not to be super paining the ass but wouldn't your convention put the doctoring after the """ :P

Reply:
We have a docstring convention? 🤣 (why yes it looks like we do)

although as a side note I would mention that the `pydocstyle` convention requires multiline docstrings to start from a new line

Reply:
I think this is what we do @piiq .

Short doctoring starts above and long (multiline) starts below.